app.js

```diff
@@ -1,16 +0,0 @@
-const express = require("express")
-const app = express() ;
```

Comment:
All imports first, constant declarations should come later 

const express = require("express")
const bodyParser = require("body-parser")
const appRoutes = require("./router/routes")
require('dotenv').config() ; 

const app = express() ;